frontend/src/charts/ChoroplethMap.tsx

```diff
@@ -116,10 +113,21 @@ export function ChoroplethMap(props: ChoroplethMapProps) {
       titleLimit: 0,
       font: "monospace",
       labelFont: "monospace",
+      labelOverlap: "greedy",
+      labelSeparation: 10,
       offset: 10,
+      format: "d",
     };
     if (props.numberFormat === "percentage") {
```

Comment:
could we switch numberFormat to MetricConfig.type? They should always be the same and it reduces chance of people forgetting to set the numberFormat properly (I think numberFormat may have been created before we added the MetricConfig)

Reply:
Sg, done